game_function.php

```diff
@@ -0,0 +1,24 @@
+
+<?php
+
+function printStats(array $result): string
+{
+
+    $game_data = '';
+
+    foreach ($result as $data) {
```

Comment:
What happens if there is no game in the database for whatever reason? Displaying an error message would be a nice touch!